src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -327,21 +356,18 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
     }
 
     return (
-      <>
-        <EuiSpacer />
-        <EuiSwitch
-          data-test-subj="saveAsNewCheckbox"
-          checked={this.state.copyOnSave}
-          onChange={this.onCopyOnSaveChange}
-          label={
-            <FormattedMessage
-              id="savedObjects.saveModal.saveAsNewLabel"
-              defaultMessage="Save as new {objectType}"
-              values={{ objectType: this.props.objectType }}
-            />
-          }
-        />
-      </>
+      <EuiSwitch
+        data-test-subj="saveAsNewCheckbox"
+        checked={this.state.copyOnSave}
+        onChange={this.onCopyOnSaveChange}
+        label={
+          <FormattedMessage
+            id="savedObjects.saveModal.saveAsNewLabel"
+            defaultMessage="Save as new {objectType}"
+            values={{ objectType: this.props.objectType }}
+          />
+        }
+      />
```

Comment:
If the user turns on the "Save as new group" switch while the title input contains the name of an existing saved annotation, would it be possible to automatically append a count to the title to avoid saved object name collisions? For example, if I attempt to save a new annotation group from an existing saved annotation group called "My annotation", turning on "Save as new group" switch would add a prefix to the title automatically, resulting in something like: "My annotation [1]". Thoughts?